L1Trigger/L1THGCal/python/egammaIdentification.py

```diff
@@ -67,30 +69,84 @@ def __init__(self, eta_min, eta_max, pt_min, pt_max):
         },
         ]
 
-# Identification for 3D HistoMax clustering (as in 3.5.2)
-bdt_weights_histomax = [
-        # Low eta
-        'L1Trigger/L1THGCal/data/egamma_id_histomax_352_loweta_v0.xml',
-        # High eta
-        'L1Trigger/L1THGCal/data/egamma_id_histomax_352_higheta_v0.xml',
-        ]
-
-working_points_histomax = [
-        # Low eta
-        {
-        '900':0.19146989,
-        '950':0.1379665 ,
-        '975':0.03496629,
-        '995':-0.24383164,
-        },
-        # High eta
-        {
-        '900':0.13347613,
-        '950':0.04267797,
-        '975':-0.03698097,
-        '995':-0.23077505,
-        },
-        ]
+# Identification for 3D HistoMax clustering: switch between configurations for v8 and v9 geometry
+input_features_histomax = {
+        "v8_352":['cl3d_firstlayer', 'cl3d_coreshowerlength', 'cl3d_maxlayer', 'cl3d_srrmean'],
+        "v9_370":['cl3d_coreshowerlength', 'cl3d_showerlength', 'cl3d_firstlayer', 'cl3d_maxlayer', 'cl3d_szz', 'cl3d_srrmean', 'cl3d_srrtot', 'cl3d_seetot', 'cl3d_spptot'],
+        "v9_394":['cl3d_coreshowerlength', 'cl3d_showerlength', 'cl3d_firstlayer', 'cl3d_maxlayer', 'cl3d_szz', 'cl3d_srrmean', 'cl3d_srrtot', 'cl3d_seetot', 'cl3d_spptot']
```

Comment:
maybe there should be a separate variable with the list of v9 variables that is used for both v9 cases in this dict (unless they're expected to become different lists in the future)